ouroboros-consensus/docs/Contributing.md

```diff
@@ -1,5 +1,35 @@
-TODO eg git history goals etc
+# How to contribute to the Consensus Layer
 
-See the
-https://github.com/input-output-hk/ouroboros-network/ouroboros-consensus/docs/StyleGuide.md
-document.
+See the follow documents.
+
+  * this repository's [README][repo-README]; it links to design documents
+  * Consensus [style guide][consensus-style]
+  * Consensus [git process][consensus-git]
+  * Consensus [onboarding][consensus-onboarding]
+  * TODO quick-start guide
```

Comment:
@EncodePanda I'm envisioning your PR #2885 document here. Do you think that's a good fit?

Reply:
yeah, I think so, let's get this PR in first, and then I will apply my changes accordingly